### PR TITLE
Restore Google Analytics

### PR DESCRIPTION
**Pull request by davetcoleman**
ROS 1 tutorials were connected to the moveit.ros.org Google Analytics account [here](https://github.com/ros-planning/moveit2_tutorials/pull/208/files#diff-796eab59177eac7e3a920bc2c60dab7dc133e977ebcaab971c80c5147d3a3a18L80)

This restores that, though its not using the fancy tag manager approach but just the built in sphynix-rtd-theme approach.

This should be cherry-picked to foxy and galactic.

### CODE REVIEW

**Comment by vatanaksoytezer** [comment]
@davetcoleman can you run `pre-commit run -a` so CI is happy, and then we can merge this.

**Review by mamoll — APPROVED** [review]
:+1: modulo minor change to make CI happy; see below.

File: conf.py [hunk 1]
```diff
@@ -105,6 +105,8 @@
     "collapse_navigation": False,
     "sticky_navigation": True,
     "navigation_depth": -1,
+    # This is the Google Analytics account used by moveit.ros.org, not picknik.ai
+    "analytics_id": 'UA-108532843-1',
```
Comment (mamoll) [inline conf.py:109]:
```suggestion
    "analytics_id": "UA-108532843-1",
```
Make CI happy.

**Review by vatanaksoytezer — APPROVED** [review]

**Comment by davetcoleman** [comment]
@skolk fyi analytics is online

**Pull request merged**